packages/components/src/truncate/test/index.tsx

```diff
@@ -10,34 +10,34 @@ import { Truncate } from '..';
 
 describe( 'props', () => {
 	test( 'should render correctly', () => {
-		const { container } = render( <Truncate>Lorem ipsum.</Truncate> );
-		expect( container.firstChild ).toHaveTextContent( 'Lorem ipsum.' );
+		render( <Truncate>Lorem ipsum.</Truncate> );
+		expect( screen.getByText( 'Lorem ipsum.' ) ).toBeInTheDocument();
```

Comment:
Except the rare cases when we're testing hidden elements, I'd always recommend using `toBeVisible()` over `toBeInTheDocument()` for visible elements, because it's a bit more specific and assures that the user actually sees it.

```suggestion
		expect( screen.getByText( 'Lorem ipsum.' ) ).toBeVisible();
```

Reply:
Thanks 👍  Good to know!